appveyor.yml

```diff
@@ -6,6 +6,7 @@ cache:
 install:
   - ps: Install-Product node $env:nodejs_version x64
   - yarn
+  - yarn run prepare
   - git submodule sync
```

Comment:
@dickeyxxx I was wondering why there was no equivalent circle.yml change and found that it does an `npm run prepare` instead, could you make that a `yarn run prepare`?  I know that does not really change anything, just trying to be consistent.